docs/collector.dfsr.md

```diff
@@ -0,0 +1,72 @@
+# dfsr collectors
+
+The dfsr collectors expose metrics for [DFSR](https://docs.microsoft.com/en-us/windows-server/storage/dfs-replication/dfsr-overview).
+
+|||
+-|-
+Metric name prefix  | `dfsr_connection`, `dfsr_folder`, `dfsr_volume`
+Data source         | Perflib
+Enabled by default? | No
+
+## Flags
+
+None
+
+## Metrics
+
+Name | Description | Type | Labels
+-----|-------------|------|-------
+`dfsr_connection_bandwidth_savings_using_dfs_replication_total` | | counter | None
```

Comment:
The code seems to have a `name` label, for most metrics, at least?

Reply:
My mistake, I'll add the label for each entry.